cf/commands/stack_test.go

```diff
@@ -39,6 +39,12 @@ var _ = Describe("stack command", func() {
 
 			Expect(testcmd.RunCommand(cmd, []string{}, requirementsFactory)).To(BeFalse())
 		})
+
+		It("fails with usage when not provided exactly one arg", func() {
+			requirementsFactory.LoginSuccess = true
+			Expect(testcmd.RunCommand(cmd, []string{}, requirementsFactory)).To(BeFalse())
+			Expect(ui.FailedWithUsage).To(BeTrue())
+		})
```

Comment:
Since you are specifically trying to address the messages returned for incorrect usages, as in your description, I think you should add assertions making sure that the messages expected are outputted.